.drone.yml

```diff
@@ -120,13 +123,49 @@ pipeline:
       event: deployment
       environment: qa
 
-  deploy-to-prod:
+  deploy-to-wcs-qa:
     image: quay.io/ukhomeofficedigital/kd:v1.16.0
     environment:
+      - KUBE_NAMESPACE=wcs-qa
+      - ENVIRONMENT=qa
+      - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+    secrets:
+      - hocs_docs_converter_wcs_qa
+    commands:
+      - source version.txt
+      - echo $VERSION
+      - echo $KUBE_NAMESPACE
+      - cd kube-hocs-docs-converter
+      - ./deploy.sh
+    when:
+      event: deployment
+      environment: qa
+
+  deploy-to-cs-prod:
+    image: quay.io/ukhomeofficedigital/kd:v1.16.0
+    environment:
+      - KUBE_NAMESPACE=cs-prod
+      - ENVIRONMENT=prod
```

Comment:
These aren't strictly required; you're duplicating the contents of the automatically populated `$DRONE_DEPLOY_TO` environment variable.

Reply:
True, and this will be changed in the next wave across all projects for qa and prod.